enterprise/api/v1/views/enterprise_customer_sso_configuration.py

```diff
@@ -59,3 +78,115 @@ def oauth_orchestration_complete(self, request, configuration_uuid, *args, **kwa
         sso_configuration_record.active = True
         sso_configuration_record.save()
         return Response(status=HTTP_200_OK)
+
+    @permission_required(
+        'enterprise.can_access_admin_dashboard',
+        fn=lambda request, *args, **kwargs: get_customer_from_request(request)
+    )
+    def retrieve(self, request, *args, **kwargs):
+        configuration_uuid = kwargs.get('configuration_uuid')
+        # Look for the configuration record
+        sso_configuration_record = self.queryset.filter(pk=configuration_uuid).first()
+        if not sso_configuration_record:
+            return Response(status=HTTP_404_NOT_FOUND)
```

Comment:
This code block of getting the sso config from the request is used in at least a few places, might be worthwhile to put in helper method.